intelmq/bots/parsers/shadowserver/config.py

```diff
@@ -889,9 +847,7 @@ def validate_ip(value):
         'classification.type': 'vulnerable service',
         'classification.taxonomy': 'Vulnerable',
         'protocol.application': 'xdmcp',
-        'feed.code': 'shadowserver-openxdmcp',
-        'feed.name': 'shadowserver',
-        'feed.url': 'https://www.shadowserver.org/wiki/pmwiki.php/Services/Open-XDMCP',
+        # 'feed.url': 'https://www.shadowserver.org/wiki/pmwiki.php/Services/Open-XDMCP',
```

Comment:
This is not feed.url, it's the documentation, which is already linked above on line 825.